convert_matrices.py

```diff
@@ -1707,11 +1707,11 @@ def main(project_dirs: List[Path]):
             else:
                 succeeded_projects.append(project_dir)
     finally:
+        skipped_projects = converter_classes.keys()
+        print_projects('not in working set', skipped_projects)
         print_projects('not implemented', not_implemented_projects, file=sys.stderr)
         print_projects('failed', failed_projects, file=sys.stderr)
         print_projects('succeeded', succeeded_projects)
-        for k, v in converter_classes.items():
```

Comment:
I think this should remain in place, because it serves an important purpose. But maybe it should be conditional on SKUNK_ACCESSIONS being unset.